docs/docsite/rst/dev_guide/developing_modules_general_aci.rst

```diff
@@ -396,6 +396,18 @@ You can run from your fork something like:
 .. code-block:: bash
 
     $ ansible-test sanity --python 2.7 lib/ansible/modules/network/aci/aci_tenant.py
+    
+For collections you need to build & install first and then run the sanity inside the installed directory:
```

Comment:
```suggestion
For collections you need to build & install first and then run the sanity inside the installed directory:
```
Hi @lumean , thanks for the PR! When developing collections, i think, in most cases it looks like:
1. Create this proper path to clone a repo to:
`mkdir -p ~/ansible_collections/<namespace>/<collection>`
2. Clone the repo:
`git clone git@github.com:... ~/ansible_collections/<namespace>/<collection>`
3. Cd there:
`cd ~/ansible_collections/<namespace>/<collection>`
4. Create a branch, change something
5. Run tests (sanity and integration if applicable) against the changed file(s) (we don't need to run sanity tests against all files in the collection that we didn't touch) and we don't need to build collections before testing:
`ansible-test sanity <path-to-changed-file> --docker`

What do you think?

Also there are sanity failings in the PR (see the comment by bot)

Reply:
Hi @Andersson007 , thanks for the feedback, I was missing exactly the workflow steps 1-3, which was not obvious to me after reading the docs for developing modules. Meanwhile I found also  https://docs.ansible.com/ansible/latest/dev_guide/developing_collections.html#contributing-to-collections   that more or less describes the steps.

I think a cross-link to that doc page would be more appropriate instead of my initial proposal here.
Since I wanted to test a module (which was part of a collection, cloned to an arbitrary directory and not `~/ansible_collections/<namespace>/<collection>`) I only read the  `developing module` docs, which doesn't go into details about different workflow if that module is part of a collection.

Reply:
@lumean to put the link sounds good to me. It's better to use the doc version from `devel` (https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#contributing-to-collections), not from `latest` because there's the most fresh version of the document in `devel` (just replace the word `latest` with `devel` in your reference).